src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb

```diff
@@ -37,75 +38,96 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
+                         End Function)
+
+            ' Import directives
             BindToOption(PlaceSystemNamespaceFirst, GenerationOptions.PlaceSystemNamespaceFirst, LanguageNames.VisualBasic)
             BindToOption(SeparateImportGroups, GenerationOptions.SeparateImportDirectiveGroups, LanguageNames.VisualBasic)
             BindToOption(SuggestForTypesInReferenceAssemblies, SymbolSearchOptions.SuggestForTypesInReferenceAssemblies, LanguageNames.VisualBasic)
             BindToOption(SuggestForTypesInNuGetPackages, SymbolSearchOptions.SuggestForTypesInNuGetPackages, LanguageNames.VisualBasic)
-            BindToOption(AddMissingImportsOnPaste, FeatureOnOffOptions.AddImportsOnPaste, LanguageNames.VisualBasic, Function()
-                                                                                                                         ' If the option has Not been set by the user, check if the option to enable imports on paste
-                                                                                                                         ' Is enabled from experimentation. If so, default to that. Otherwise default to disabled
-                                                                                                                         If experimentationService Is Nothing Then
-                                                                                                                             Return False
-                                                                                                                         End If
-
-                                                                                                                         Return experimentationService.IsExperimentEnabled(WellKnownExperimentNames.ImportsOnPasteDefaultEnabled)
-                                                                                                                     End Function)
+            BindToOption(AddMissingImportsOnPaste, FeatureOnOffOptions.AddImportsOnPaste, LanguageNames.VisualBasic,
+                         Function()
+                             ' If the option has Not been set by the user, check if the option to enable imports on paste
+                             ' Is enabled from experimentation. If so, default to that. Otherwise default to disabled
+                             If experimentationService Is Nothing Then
+                                 Return False
+                             End If
+
+                             Return experimentationService.IsExperimentEnabled(WellKnownExperimentNames.ImportsOnPasteDefaultEnabled)
+                         End Function)
+
+            ' Highlighting
+            BindToOption(EnableHighlightReferences, FeatureOnOffOptions.ReferenceHighlighting, LanguageNames.VisualBasic)
+            BindToOption(EnableHighlightKeywords, FeatureOnOffOptions.KeywordHighlighting, LanguageNames.VisualBasic)
```

Comment:
i moved the initialization around to match the order it appears in tools|options.  it was super hard to figure out where things needed to go otherwise.